pints/tests/test_toy_fitzhugh_nagumo_model.py

```diff
@@ -59,6 +60,16 @@ def test_values(self):
         self.assertAlmostEqual(values[200, 0], 1.675726, places=6)
         self.assertAlmostEqual(values[200, 1], -0.226142, places=6)
 
+    def test_sensitivities(self):
+        model = pints.toy.FitzhughNagumoModel([2, 3])
+        parameters = [0.2, 0.7, 2.8]
+        times_finer = np.linspace(0, 20, 500)
+        sols, sens = model.simulateS1(parameters, times_finer)
+        f = interp1d(times_finer, sens[:, 0][:, 2])
```

Comment:
What's going on here???

Reply:
I've added comments to these -- to obtain the sensitivity at a predefined time it's easiest to interpolate the sensitivity series.

Reply:
I don't understand. Why don't you request evaluation at the times you want?

Reply:
I could but find it easier to do it this way as it means I don't have to keep tabs on which observation maps to which time -- feel free to change if you feel strongly though.

Reply:
Made it easier, but found a bug!